app/globals.css

```
@@ -104,117 +222,56 @@
   }
 }
 
-@layer base {
-  h1 {
-    font-family: Poppins;
-    font-size: 4rem;
-    font-style: normal;
-    font-weight: 700;
-    line-height: 130%;
-    letter-spacing: -0.04rem;
-  }
-}
-
-@layer base {
-  h2 {
-    text-align: center;
-    font-family: Poppins;
-    font-size: 3rem;
-    font-style: normal;
-    font-weight: 700;
-    line-height: 115%;
-    letter-spacing: -0.03rem;
-  }
-}
-
-@layer base {
-  h3 {
-    text-align: center;
-    font-family: Poppins;
-    font-size: 2rem;
-    font-style: normal;
-    font-weight: 600;
-    line-height: 105%;
-    letter-spacing: -0.01rem;
-  }
-}
-
-@layer base {
-  .regbase {
-    font-family: Poppins;
-    font-size: 1.5rem;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 130%;
-  }
-}
+/* Skills */
 
 @layer base {
-  .boldbase {
-    font-family: Poppins;
-    font-size: 1.5rem;
-    font-style: normal;
-    font-weight: 600;
-    line-height: 130%;
-  }
-}
-
-@layer base {
-  .regp {
-    font-family: Poppins;
-    font-size: 1.25rem;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 150%;
-  }
-}
-
-@layer base {
-  .boldp {
-    font-family: Poppins;
-    font-size: 1.25rem;
-    font-style: normal;
-    font-weight: 600;
-    line-height: 130%;
+  .skills {
+    display: flex;
+    width: 21.5625rem;
+    flex-direction: column;
+    align-items: center;
+    gap: 2.5rem;
   }
 }
 
 @layer base {
-  .regbody {
-    font-family: Poppins;
-    font-size: 1.125rem;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 160%;
+  .skillslg {
+    display: flex;
+    width: 90rem;
+    padding: 4.5rem 5.3125rem;
+    flex-direction: column;
+    align-items: flex-start;
```

```suggestion
    align-items: center;
```
This is why the text isn't centered. Flex-start will place it at the start of the flexbox.

---

That'll do it!